content/02.introduction.md

```diff
@@ -47,46 +46,96 @@ The transmission and mortality rate estimations of COVID-19 remain to be determi
 
 Among known coronaviruses, SARS-CoV-2 has the closest overall sequence similarity to RaTG13 (~96%) found in a *Rhinolophus affinis* bat [@doi:10.1038/s41586-020-2012-7], while the receptor binding domain (RBD) is highly similar to that of viruses found in pangolins [@doi:10.1101/2020.02.19.950253].
 This suggests that SARS-CoV-2 may have originated in viral reservoirs of similar hosts; however, current evidence cannot discriminate an origin of the virus before or after zoonotic transfer to humans [@doi:10.1038/s41591-020-0820-9].
 
-### COVID-19: Mechanisms and Presentation
-
-#### Immunological Mechanisms of Coronavirus-driven Disease in Humans
-
-Coronaviruses are known to cause respiratory illnesses in humans through the following possible mechanisms...[Summarize relevant mechanisms for cell entry & address evidence for/against ACE2 being important, etc.]
+The viral spike protein, _S_, plays a critical role in viral replication.
+There are two domains: the N-terminal (S1 )domain used for binding and the C-terminal (S2) domain for fusion [@doi:10.3390/v4061011].
+The S1 domain evolves more rapidly than the S2 domain, and different coronaviruses bind to difference receptors.
+Betacoronaviruses are known to bind to CEACAM1, Neu 5,9 Ac2, and ACE2 [@doi:10.3390/v4061011].
+
+### COVID-19: Infection and Presentation
+
+#### Pathogenic Mechanisms of Coronaviruses
+
+Until more information becomes available about the pathogenesis of SARS-CoV-2 specifically, insight can be gained by examining how the human body responds to viral pathogens generally, how infection from related viruses proceeds, and how this information intersects with what is known about COVID-19.
+Following exposure to a pathogen such as a virus, pathogenesis involves three major components: entry, replication, and spread [@isbn:0-9631172-1-1 {chap. 45}]. 
+The human immune response is double-pronged, with the innate and the adaptive immune response.
+Through innate immunity, the body can rapidly recognize and attack pathogens and infected cells. Cells from innate immunity also pass critical information to prime the adaptive immune response.
+Adaptive immunity can be broadly classified into humoral and cell-mediated responses. 
+These are unique from innate immune responses because they target specific protein sequences from pathogens. 
+The humoral response includes soluble proteins and antibodies that can neutralize and eliminate pathogens in circulation that have not yet entered cells, and can also mediate killing of pathogen-infected cells. 
+Cell-mediated responses include the killing of pathogen-infected cells by cytotoxic CD8+ T cells, and the secretion of cytokines to drive inflammation and support immune cell activities.
+The antibody-mediated response works to prevent or reduce entry into cells, while the cell-mediated response works to stop the pathogen's replication and spread.
+
```

Comment:
Suggested transition sentences & also informational sentences before going into (subsection about) specifics which include SARS-CoV-2 research (and b/c on line 96 you say that you're starting with other information):

There are a few reviews on the research coming out specific to SARS-CoV-2 with respect to the interferon-mediated innate antiviral responses which are initiated by the virus-infected cells themselves but can also sometimes have inflammatory and systemic effects [doi: 10.1016/j.chom.2020.05.008], the innate immune system response with the focus being on innate immune cells and responses other than the interferon-mediated responses [doi: 10.1016/j.chom.2020.05.009], and different SARS-CoV-2-specific Immunoglobulins (as well as viral RNA) detected over the course of an infection [doi: 10.3389/fimmu.2020.00879].
Next, we discuss some particular aspects of SARS-CoV-2 pathogenesis and host immune response based on primary research, reviews, and context from other viruses.

Reply:
I did not find/read yet a fully-processed(non-preprint) review of acquired immune defenses related to SARS-COV-2 (focusing on the initial exposure / not focusing on vaccine stuff) but I did find/read a good review for T-cells and SARS (CoV-1) from 2014 & there are some articles (non-reviews) which mention acquired immunity and also one of the innate reviews mentions T-cells a bit. For now, in order to also mention T-cells, after/with the list of reviews above we could have a sentence (or comma-clause or semi-colon clause) like:
A review of T-cell mediated responses to respiratory coronaviruses notes a large reduction in T-cells associated with the acute phase of SARS (from SARS-CoV-1) [doi: 10.1007/s12026-014-8534-z] and so far some articles and reviews on SARS-CoV-2 also mention some evidence for reduction in the number of certain T-cells as well as NK-cells [doi: 10.1016/j.chom.2020.04.009; doi: 10.1016/j.chom.2020.05.009].

Reply:
for my above T-cell comment above would someone (@gregszetoAI for example?) confirm that I'm write to make the possible connection between SARS-CoV-1 and SARS-CoV-2 observations on reduction in T-cells (just checking b/c there are different types of T-cells & the timing also matters & I don't regularly look at the immune system at this particular level/scale/etc.)

Reply:
@rando2 I noticed that some other reviews that cover complex topics (like immune system or more often subtopics of immune system) cite other reviews which cover related or complementary topics/subtopics. That's why I suggested maybe mentioning together somewhere major reviews (e.g., from nature or cell journals) focused on subtopics of immune system in case it might be useful to readers who later may want to look these up.